### DISCUSSION TITLE
Configuration of the Docker Container

### DISCUSSION THREAD

**Discussion by FelixStumvoll** [OrchardCMS/OrchardCore#15869] [Q&A]
i recently found this [docker image](https://hub.docker.com/r/orchardproject/orchardcore-cms-linux) and it has only a parameter for the App_Data Path. 
How is it intended to configure this container, my solution for now is to copy a configuration with a customer Dockerfile. 
Are there any alternatives or has anyone solved this in a different more elegant way?

**Comment by vitalybrandes** [comment]
> i recently found this [docker image](https://hub.docker.com/r/orchardproject/orchardcore-cms-linux) and it has only a parameter for the App_Data Path.
> How is it intended to configure this container, my solution for now is to copy a configuration with a customer Dockerfile.
> Are there any alternatives or has anyone solved this in a different more elegant way?

Can you explain more about your question?

**Comment by Skrypt** [comment]
I have a PR here that fixes the different files : 
https://github.com/OrchardCMS/OrchardCore/pull/7077
Our Dockerignore and Dockerfile is meant for building on the CI only.
In that PR I've built a proper Dockerignore and Dockerfile for everyone to be able to build OC with Docker from any OS directly.
Though, the appropriate way to deploy with Docker is to use `docker-compose`.
You can see an example of that in that PR which I'm not done with : https://github.com/OrchardCMS/OrchardCore/pull/7085/files#diff-4e5e90c6228fd48698d074241c2ba760
